### DISCUSSION TITLE
Links from within PDF  ?

### DISCUSSION THREAD

**Discussion by r-curry** [RyotaUshio/obsidian-pdf-plus#190] [Q&A]
Apologies if this has been addressed elsewhere, I've looked through the documentation and haven't found anything.

What I am trying to do is essentially,
- Inside a PDF [[PDF 1]] I create an annotation and remark that this sentence is related to [[Some other page]].
- What I want to happen, is then later when I go to [[Some other page]], I'd like the link from [[PDF 1]] to appear in the backlinks pane. 
- Currently this doesn't seem to work.

Is there a setting I can play with to make this happen? Any tips very much appreciated!

**Comment by N3C2L** [comment]
If you create internal pdf links from one pdf page to another, that alone won't be visible in the backlink pane. But you can create a copy link formats inside the pdf++ settings that pointing to your 2 connected pages. You can then jump from this or that link via the markdown file.

**Reply by r-curry** [reply]
Hello. I really appreciate your suggestion, but I am not sure I quite understand it. 

Regarding your first point about the markdown file being visible. I would assume the backlines pane looks through all the other .md files in the vault and if the title of the currently open file appears then it lists those in the backlines pane. I am just thinking it would be great if it could also look through the annotations created within pdf files. If that makes sense.

**Reply by N3C2L** [reply]
Yeah, I know what you mean and I wasn't clear enough.
What's happening when you are clicking on a link in the backlink pane?
It takes you to your markdown file.
That's why you can create 2 copy link formats:
One named "Link Source" + Color Name "Link Source".
Second name "Link Destination" + Color Name "Link Destination".
You can then select a line or a single word in your text and choose one of the color/copy link formats from above. These 2 formats then appear in the backlink sections of your PDF if you have linked them to your markdown file. Double click one of your highlights to jump directly to the link in the markdown file (without the need to look at your backlink pane). You could paste that link in your browser and create your copy link formats: obsidian://pdf-plus?setting=copyCommands

**Reply by matar3** [reply]
@N3C2L That's a good idea, and i opened a FR about it !
See here https://github.com/RyotaUshio/obsidian-pdf-plus/issues/230
